php-packages/testing/src/integration/Extension/ExtensionManagerIncludeCurrent.php

```diff
@@ -112,4 +104,32 @@ protected function getAssetsFilesystem(): Cloud
     {
         return new FilesystemAdapter(new FlysystemFilesystem(new Local($this->paths->public.'/assets'), ['url' => resolve('flarum.config')->url().'/assets']));
     }
+
+    protected function includeCurrentExtension($package, string $packagePath)
+    {
+        if (Arr::get($package, 'type') === 'flarum-extension') {
+            $current = new Extension($packagePath, $package);
+            $current->setInstalled(true);
+            $current->setVersion(Arr::get($package, 'version'));
+            $current->calculateDependencies([], []);
+
+            $this->extensions->put($current->getId(), $current);
+
+            $this->extensions = $this->extensions->sortBy(function ($extension, $name) {
+                return $extension->composerJsonAttribute('extra.flarum-extension.title');
+            });
```

Comment:
Should this be using top-sort instead? I don't remember why this was added here.

Reply:
Can't recall, let's leave it be for now.